pydrive2/drive.py

```diff
@@ -47,3 +47,28 @@ def GetAbout(self):
         :returns: A dictionary of Google Drive information like user, usage, quota etc.
         """
         return self.auth.service.about().get().execute(http=self.http)
+        
+
+    @LoadAuth
+    def CopyFile(self, source_file_id, target_folder_id, new_name):
```

Comment:
I this it should be part of the `files.py` and `GoogleDriveFile` class. 

It won't have `source_file_id` then, only target path (folder) and new name. Not yet sure if we want to accept `target_folder_id` as an id or as another `GoogleDriveFile` instance (I would probably do latter) + regular things like `params`, etc that are part of the interface.

also, when we agree on the interface, let's please add a test for this